frontend/app/assets/javascripts/spree/frontend/product.js

```diff
@@ -78,3 +80,49 @@ Spree.ready(function ($) {
 
   return Spree.addImageHandlers()
 })
+
+Spree.ready(function ($) {
```

Comment:
No jQuery 🎉 is used in this function, so no need for `$`
```suggestion
Spree.ready(() => {
```